torch/ao/ns/fx/n_shadows_utils.py

```diff
@@ -593,16 +523,14 @@ def handle_subgraph_candidate(
         setattr(mt, attr_name, orig_mod_copy_wrapped)
 
         # add a call to the wrapper module from the parent graph
-        with mt.graph.inserting_after(last_node):
+        insert_after_node = last_added_shadow_node_list[0]
+        with mt.graph.inserting_after(insert_after_node):
             # TODO(future PR): handle fusion patterns where non-first nodes
             # need inputs
 
             # pass in all node args and kwargs
 
-            # the first argument is always the reference output of the last
-            # node of this subgraph
-            new_args = [last_node]
-
+            new_args = []
```

Comment:
just for my understanding, why is the `last_node` argument removed?

Reply:
Before it was

```
# parent
x0 -> op -> x1 
  \                    \
    -----------shadow

# shadow
x0 -> op_shadow -> x1_shadow -> log 
                                                            /
x1 --------------------------------
```

Now, it's

```
# parent
x0 -> op -> x1 ---------> log
  \                                         /  
    -shadow -> x1_shadow


# shadow
x0 -> op_shadow -> x1_shadow
```

the logger is moved outside, and `last_node` is the thing which the logger needs to compare actual to reference